config_src/drivers/unit_tests/test_MOM_EOS.F90

```diff
@@ -0,0 +1,10 @@
+program test_MOM_EOS
+
+use MOM_EOS, only           : EOS_unit_tests
+use MOM_error_handler, only : set_skip_mpi
+
+call set_skip_mpi(.true.) ! This unit tests is not expecting MPI to be used
+
+if ( EOS_unit_tests(.true.) ) stop 9
```

Comment:
Why not `error stop`? stop can imply a normal exit (albeit with an abnormal exit code).

Reply:
Because it causes a backtrace and that would not be helpful (even inappropriate) for a unit test.

Reply:
ok I understand. (9 is a bit arbitrary but thats fine.)

Reply:
Change to 1?

Reply:
I would probably just use `stop` and let the compiler decide, but I think it's not too important.